### PR TITLE
Update travis matrix to current versions (go, ubuntu, zfs)

### PR DESCRIPTION
**Pull request by thaJeztah**
See individual commits for details

### CODE REVIEW

**Comment by thaJeztah** [comment]
Looks like travis integration is disabled currently 🤔 @Mic92

**Comment by Mic92** [comment]
Maybe travis needs to be reactivated. There was a switch from travis.org to travis.com at some point.

**Comment by thaJeztah** [comment]
Yes, having travis reactivated would be good; I made this change based on "checking what's current", but have not been able to test this. Thought I'd open a PR for it for consideration (hoping someone would be able to reactivate travis)

**Comment by Mic92** [comment]
I think travis also runs in your own repository if you have enabled it there.

**Comment by thaJeztah** [comment]
Oh, good one; never tried; I can try that

File: .travis.yml [hunk 1]
```diff
@@ -26,8 +28,8 @@ before_install:
   - sudo apt-get update -y && sudo apt-get install -y libattr1-dev libblkid-dev linux-headers-$(uname -r) tree uuid-dev
   - mkdir -p $HOME/zfs
   - cd $HOME/zfs
-  - [[ -d spl-$rel.tar.gz ]] || curl -L https://github.com/zfsonlinux/zfs/releases/download/zfs-$rel/spl-$rel.tar.gz | tar xz
-  - [[ -d zfs-$rel.tar.gz ]] || curl -L https://github.com/zfsonlinux/zfs/releases/download/zfs-$rel/zfs-$rel.tar.gz | tar xz
+  - [[ -d spl-$rel.tar.gz ]] || curl -L https://github.com/openzfs/zfs/releases/download/zfs-$rel/spl-$rel.tar.gz | tar xz
```
Comment (Mic92) [inline .travis.yml:31]: Spl was removed. It was merged into zfs. If you drop `0.6.5.11` than it is not needed anymore.

Comment (thaJeztah) [reply .travis.yml:31]: Yes, might be good to do that; also looks like travis is failing on this line (invalid yaml?), so it'd be killing two birds with one stone

Comment (thaJeztah) [reply .travis.yml]: oh, actually, it's still needed for v0.7 looks like

Comment (Mic92) [reply .travis.yml]: It might be time to kill v0.7 though.

Comment (thaJeztah) [reply .travis.yml]: Let me try spending some time on this later on; I tried changing but things failed (got travis working on my branch); probably made some mistake. Thanks!

**Comment by mmlb** [comment]
Closing in favor of GitHub Actions in #79

**Pull request closed without merging**